packages/block-library/src/quote/edit.js

```diff
@@ -85,33 +85,37 @@ export default function QuoteEdit( {
 					</ToolbarButton>
 				</ToolbarGroup>
 			</BlockControls>
-			<BlockQuotation { ...innerBlocksProps }>
-				{ innerBlocksProps.children }
+			<figure { ...innerBlocksProps }>
+				<BlockQuotation>{ innerBlocksProps.children }</BlockQuotation>
```

Comment:
`innerBlocksProps` must be passed to the parent element.

Reply:
This will be problematic... :thinking: If the props are in the `<blockquote>` element, then themes won't be able to add the left border that we currently use - which spans both the quote and the citation.
Any ideas on how to address that? Styling should be applied on the parent element (in this case the `<figure>`, which is why the classes are getting applied there.
I tried moving the `innerBlockProps` to the blockquote, but that was messing up styles a lot. Is there a problem with them being in the figure instead?